apps/rig_inbound_gateway/lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -84,12 +110,22 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
 
     receive do
       {:response_received, response} ->
+        ProxyMetrics.count_proxy_request(conn.method, request_path, "http", response_from, "ok")
```

Comment:
is the request_path different from `conn.request_path`?

Reply:
Hmm - Yes and no. The `request_path` is actually passed from the parametrization. 

But I think the HTTPHandler is only called if the `request_path` from param & the `conn.request_path` are matching (If i understood the code correctly)

Hence - It's the same. I didn't see anywhere that the `conn `was updated...

Reply:
Do you think I should use `conn.request_path` everywhere? At least the code would shrink again, as I don't have to pass request_path everywhere

Reply:
a request path may also be matched using a regular expression. To summarize, you have the following choices:

- using `conn.request_path`, we'd see what was actually requested by the client
- using the definition's request path or regex pattern, we'd see what endpoint path patterns have matched
- using the endpoint definition's ID, we'd see what endpoints definitions matched in general

Reply:
As discussed - `conn.request_path `is probably the best option as this is the only option also usable during `not_found`. To have a consitent view I'll use that